src/plugins/query_enhancements/public/query_assist/components/query_assist_summary.tsx

```diff
@@ -0,0 +1,286 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import {
+  EuiSplitPanel,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiIcon,
+  EuiIconTip,
+  EuiSmallButtonIcon,
+  EuiSpacer,
+  EuiCopy,
+} from '@elastic/eui';
+
+import React, { useEffect, useState } from 'react';
+import { i18n } from '@osd/i18n';
+import { IDataFrame } from 'src/plugins/data/common';
+import { v4 as uuidv4 } from 'uuid';
+import { isEmpty } from 'lodash';
+import { merge, of } from 'rxjs';
+import { filter, distinctUntilChanged, mergeMap } from 'rxjs/operators';
+import { HttpSetup } from 'opensearch-dashboards/public';
+import { useQueryAssist } from '../hooks';
+import { DataPublicPluginSetup, QueryEditorExtensionDependencies } from '../../../../data/public';
+import { UsageCollectionSetup } from '../../../../usage_collection/public';
+import { CoreSetup } from '../../../../../core/public';
+import sparkleHollowSvg from '../../assets/sparkle_hollow.svg';
+
+export interface QueryContext {
+  question: string;
+  query: string;
+  queryResults: any;
+}
+
+export interface QueryAssistSummaryProps {
```

Comment:
do we have a types file? if not should we add one localized to query_assist and export these update there

Reply:
i think this is only exported for testing, i usually just use `ComponentProps<typeof QueryAssistSummary>` in tests and keep this not exported